woocommerce/woocommerce-android
# Track event when change payment provider row is clicked

Closes: #6632 


### Description

This PR tracks the event when the change payment provider row is clicked.

### Testing instructions

1. Install both WCPay and Stripe extension plugins on your site
2. Navigate to Settings -> In-Person Payments
3. Click on the "Change Payment Provider" row
4. Ensure that `settings_card_present_select_payment_gateway_tapped` event is tracked in the logs
